pkg/controllers/resources/nodes/fake_syncer.go

```diff
@@ -240,3 +242,20 @@ func filterOutDaemonSets(pl *corev1.PodList) []corev1.Pod {
 
 	return podsNoDaemonSets
 }
+
+// Filter away physical DaemonSet Pods using annotations to enable scale down
+func filterOutPhysicalDaemonSets(pl *corev1.PodList) []corev1.Pod {
+	var podsNoDaemonSets []corev1.Pod
+
+	for _, item := range pl.Items {
+
+		annotations := item.GetAnnotations()
+		ownerSetKind, exists := annotations[translate.OwnerSetKind]
```

Comment:
This could be re-written as
```
if kind, ok := annotations[translate.OwnerSetKind]; ok && kind != "DaemonSet" {
	podsNoDaemonSets = append(podsNoDaemonSets, item)
}
```

Reply:
You are absolutely right :+1:  Done!